iroha_data_model/src/isi.rs

```diff
@@ -141,12 +141,12 @@ where
 
 /// Generic instruction for a removal of an object from the identifiable destination.
 #[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode)]
-pub struct Remove<D, O>
+pub struct Substract<D, O>
 where
     D: Identifiable,
     O: Value,
 {
-    /// Object which should be removed.
+    /// Object which should be substractd.
```

Comment:
```suggestion
    /// Object which should be subtracted.
```
Please check this throughout the PR